### DISCUSSION TITLE
API Integration

### DISCUSSION THREAD

**Discussion by Jayden-Htn** [chirawatj/WEB601-Assessment1#2] [General]
@chirawatj I have just added the basic API JS code for the server and routes. Files have also been reorganised. This has been pushed to dev. 

Please do the following:
- Pull these changes
- Do the commenting task assigned
- Update the project tasks on completion
- Respond to this discussion with an update on progress
- Keep progressing on other assigned tasks

**Comment by Jayden-Htn** [comment]
@chirawatj I have fixed the router functionality to navigate between pages. Please comment on these express files as tasked previously. Next meeting I would like to discuss the implementation of the CRUD methods. Please give me an update on your progress and estimated completion date.

**Comment by chirawatj** [comment]
I have added the comments to those files.
